src/firefly/js/ui/FieldGroup.jsx

```diff
@@ -30,9 +56,18 @@ export class FieldGroup extends PureComponent {
     }
 
     componentWillMount() {
-        const {groupKey:wrapperGroupKey}= this.context;
-        var {groupKey, reducerFunc, keepState, initValues, actionTypes}= this.props;
-        dispatchMountFieldGroup(groupKey, true, keepState, initValues, reducerFunc, actionTypes, wrapperGroupKey);
+        const {groupKey}= this.props;
+        const groupState= getFieldGroupState(groupKey);
+        if (groupState && groupState.mounted) {
```

Comment:
Should we also check for !keepMounted here? Looks like we never use this property, but if it is true, we will never dispatch unmount, hence the callback will never execute.